shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -967,7 +986,30 @@ - (void)onAccessibilityStatusChanged:(NSNotification*)notification {
 
   self.semanticsEnabled = enabled;
 }
+- (void)handleAccessibilityEvent:(NSDictionary<NSString*, id>*)annotatedEvent {
+  NSString* type = annotatedEvent[@"type"];
+  if ([type isEqualToString:@"announce"]) {
+    NSString* message = annotatedEvent[@"data"][@"message"];
+    NSNumber* assertiveness = annotatedEvent[@"data"][@"assertiveness"];
```

Comment:
Does this crash if `data`, `message`, or `assertiveness` properties are missing? A malformed message should be ignored (though an error should be logged ideally)

Reply:
These will result in `nil`.

Reply:
Ah OK perfect! :)